src/BrazeMessage.test.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { buildBrazeMessageComponent } from './BrazeMessage';
+
+describe('BrazeMessage', () => {
+    it('renders the correct component when a valid componentName is passed', () => {
+        const ExampleComponent = jest.fn(() => null);
+        const mappings = {
+            ExampleComponent,
+        };
+        const BrazeMessage = buildBrazeMessageComponent(mappings);
+        const componentName = 'ExampleComponent';
```

Comment:
I'd consider inlining this const in both tests. It's only used once, and it is the key part of what is being tested, so a shame if it vanishes into the noise above? Super minor point and I don't think it matters either way - up to you!

Reply:
Good point! The reason I'd pulled it out was to give it a name because like you said it's really central to the test. But given that when it's passed as a prop, it also gets that name, I think your suggestion to inline is a good one. I'll make the change :+1: